src/prefect/flows.py

```diff
@@ -733,7 +733,7 @@ def my_other_flow(name):
     @sync_compatible
     async def serve(
         self,
-        name: str,
+        name: str = None,
```

Comment:
for the type checkers

```suggestion
        name: Optional[str] = None,
```